pkg/commands/refresh.go

```diff
@@ -21,14 +29,27 @@ const (
 	deleted   refreshStatus = "Deleted"
 )
 
+const (
+	refreshInterval = time.Minute
```

Comment:
please make the default time to be 5min

Reply:
got it, shall we add a flag to enforce refresh, just for dev convenience?

Reply:
> shall we add a flag to enforce refresh, just for dev convenience?

Sure, but users may not be aware of that. That could be very useful for development and demo.